modules/miner/splitsetheap_test.go

```diff
@@ -0,0 +1,450 @@
+package miner
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// TestMapHeapSimple test max-heap and min-heap versions of the MapHeap on the
+// same sequence of pushes and pops. The pushes are done in increasing value of
+// averageFee (the value by which elements are compared).
+func TestMapHeapSimple(t *testing.T) {
+	max := &mapHeap{
+		selectID: make(map[splitSetID]*mapElement),
+		data:     make([]*mapElement, 0),
+		size:     0,
+		minHeap:  false,
+	}
+
+	min := &mapHeap{
+		selectID: make(map[splitSetID]*mapElement),
+		data:     make([]*mapElement, 0),
+		size:     0,
+		minHeap:  true,
+	}
+
+	max.Init()
+	min.Init()
+
+	for i := 0; i < 1000; i++ {
```

Comment:
for a more valid test, you want to add these elements randomly.

you can import the `github.com/NebulousLabs/fastrand` package and use the `Perm` function. `Perm(1000)` will give you 1000 integers in the range [0,1000), but they will be in random order. Then you can range through that list to get random heap insertion here.